oap-server/server-starter/src/main/resources/ui-initialized-templates/menu.yaml

```diff
@@ -0,0 +1,107 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# menus: the menus of the left navigation bar
+#   - title: the title of the menu item
+#     icon: icon name
+#     layer: related layer name, the menu item should be activated when layer has services
+#     menus: sub menus of the current item
+
+menus:
+  - title: GeneralService
```

Comment:
Title is for display, it should not be like this. `General Service` is proper value.